src/style.css

```diff
@@ -1,3 +1,52 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
-    background-color: bisque;
-}
\ No newline at end of file
+  background-color: bisque;
+}
+
+.logo {
+  width: 60px;
+  height: 60px;
+}
+
+.logo-wrap {
+  display: flex;
+}
```

Comment:
- [ ] you can make the logo text stay aligned with the image by adding `align-items: center;` after the `display: flex;` property